doc/man3/OSSL_STORE_open.pod

```diff
@@ -122,7 +121,7 @@ See L<passphrase-encoding(7)> for further information.
 
 =head1 RETURN VALUES
 
-OSSL_STORE_open() returns a pointer to a B<OSSL_STORE_CTX> on success, or
+OSSL_STRE_open() returns a pointer to a B<OSSL_STORE_CTX> on success, or
```

Comment:
wrong edit or it is just a bookmark ;)

Reply:
Strangely in my copy of the file this glitch is not present.